tests/basic/backup_test.go

```diff
@@ -1956,24 +1956,28 @@ func validateBackupCluster() bool {
 	ns := backup.GetPxBackupNamespace()
 	pods, err := core.Instance().GetPods(ns, labelSelectors)
 	if err != nil {
-		logrus.Warnf("Unable to fetch the pods from backup namespace\n Error : [%v]\n",
+		logrus.Errorf("Unable to fetch the pods from backup namespace\n Error : [%v]\n",
 			err)
 		return false
 	}
 	for _, pod := range pods.Items {
 		matched, _ := regexp.MatchString("^pxcentral-post-install-hook", pod.GetName())
-		if !matched {
```

Comment:
@snigdha-px Imagine due to some issue in the product, other pods are not in running state but pxcentral-post-install-hook pod is in completed state. With this scenario, this logic doesn't work. Its better to keep code to validate all the pods status in backup cluster and verify the expected state based on pod.

Reply:
Done